tests/test_workspace_manager_preflight.py

```
@@ -0,0 +1,316 @@
+        returncode = 0
+        async def communicate(self):
+            return b"", b""
+
+    async def fake_create(*args, **kwargs):
+        captured_args.extend(args)
+        return FakeProc()
+
+    monkeypatch.setattr(asyncio, "create_subprocess_exec", fake_create)
+    mgr = WorkspaceManager(base_root=tmp_path)
+    await mgr.repo_access_preflight(
+        "https://github.com/example/repo.git", token="ghp_TOKEN"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "token"

(S106)